tests/modules/cnvpytor/importreaddepth/test.yml

```diff
@@ -0,0 +1,10 @@
+- name: cnvpytor importreaddepth test_cnvpytor_importreaddepth
+  command: nextflow run tests/modules/cnvpytor/importreaddepth -entry test_cnvpytor_importreaddepth -c tests/config/nextflow.config
+  tags:
+    - cnvpytor
+    - cnvpytor/importreaddepth
+  files:
+    - path: output/cnvpytor/test.pytor
+      contains: '[ # TODO nf-core: file md5sum was variable, please replace this text with a string found in the file instead ]'
```

Comment:
Hmm, here you should paste a line of the `test.pytor` and remove the TODO

Reply:
So I had modified the 'contains' variable and rerun the test. It got replace by md5sum variable. 
fixed in 9c045501aefcc31d722fcdd64e4e7206de8f876e